sr_ur_launch/launch/right_arm.launch

```diff
@@ -0,0 +1,23 @@
+<launch>
```

Comment:
There is no left_arm one... We could either move both from sr_ur_bringup or just leave them there...

TBH, nothing anywhere I can find even uses sr_ur_launch for anything... we could just delete the whole package I think.